packages/perseus-editor/src/widgets/__docs__/label-image-editor.stories.tsx

```diff
@@ -81,27 +48,5 @@ class WithState extends React.Component<Empty, State> {
                 y: 78.8,
             },
         ],
-    };
-
-    render(): React.ReactNode {
-        return (
-            <div className={css(styles.wrapper)}>
-                <LabelImageEditor
-                    {...this.state}
-                    onChange={(props) =>
-                        this.setState({
-                            ...this._widget.serialize(),
-                            ...props,
-                        })
-                    }
-                    // @ts-expect-error [FEI-5003] - TS2322 - Type 'LabelImageEditor | null' is not assignable to type 'LabelImageEditor'.
-                    ref={(widget) => (this._widget = widget)}
-                />
-            </div>
-        );
-    }
-}
-
-export const Default = (args: StoryArgs): React.ReactElement => {
-    return <WithState />;
+    },
```

Comment:
Why do some of these have a `Default` and a `Preview`, but some of them only have the `Default`?